requirements.md

```diff
@@ -51,4 +51,26 @@ to describe multiple issues that may arise>
   3. ...
 2. Use Case Name (Should match functional requirement name)
    ...
+3. Add item to seller store
+   **Pre-condition:** User must be logged in
+   
+   **Trigger:** Customer selects " add items to shop" option
 
+   **Primary Sequence:** 
+   1:System allows user to place item into shop
+   2:Customer adds description of item 
+   3:Customer sets price of item
+   4:System lists item on website for sale
+   5:Customer logs out
+
+   **Alternate Sequence:**
+   2.Customer sells illegal item
```

Comment:
I don't think we should include this since it seems unnecessary and we'd have to somehow program this too.